interpretation/deepseismic_interpretation/dutchf3/data.py

```diff
@@ -441,11 +439,10 @@ class PatchLoader(data.Dataset):
     :param bool debug: enable debugging output
     """
 
-    def __init__(
-        self, config, is_transform=True, augmentations=None, debug=False,
-        ):
+    def __init__(self, config, is_transform=True, augmentations=None, debug=False, split="train"):
```

Comment:
can we put "split" before "is_transform" right after "config"? Makes more sense that way, it's more important